nums/numpy/api.py

```diff
@@ -27,12 +27,29 @@
 # pylint: disable = redefined-builtin, too-many-lines
 
 
-def _not_implemented(fun):
+def _not_implemented(func):
     # From project JAX: https://github.com/google/jax/blob/master/jax/numpy/lax_numpy.py
     def wrapped(*args, **kwargs):
         # pylint: disable=unused-argument
         msg = "NumPy function {} not yet implemented."
-        raise NotImplementedError(msg.format(fun))
+        raise NotImplementedError(msg.format(func))
+
+    return wrapped
+
+
+# TODO(mwe): Convert this to invoke the NumPy op on a worker instead of the driver
+def _default_to_numpy(func):
+    def wrapped(*args, **kwargs):
```

Comment:
Can we add this warning here: "Operation %s not implemented, falling back to NumPy. If this is too slow or failing, please open an issue on github."

Reply:
Okay.